src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -60,8 +60,8 @@ export class WebDeviceInputSystemImpl implements IDeviceInputSystem {
     private _isUsingFirefox = navigator && navigator.userAgent && navigator.userAgent.indexOf("Firefox") !== -1;

```

Comment:
Not sure this is the right way to check for navigator existence.

Reply:
changed